socialmedia.md

```diff
@@ -1,20 +1,118 @@
-# Open Source Social Media Policy v0.1
+# Social Media Policy
```

Comment:
I think we should add a version number to the document, which we can update each time we create a release, this will make it easier for people to know which version of the document they are reading.